code/src/index.css

```diff
@@ -5,9 +5,163 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+   background-color: #FFFD37
+}
+.animals{
+  width: 100%;
+  height: 50%;
 }
-
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.StartPage{
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  justify-content: center;
+  background-color: white;
+  margin: 40px;
+  padding: 20px;
+}
+
+.start-container{
+  margin: 40px;
+}
+.start-container h1{
+  font-size: 40px;
+  padding-bottom: 20px;
+  
+ 
+}
+.startBtn{
+  border-radius: 10px;
+  padding: 5px;
+  width: 150px;
+  font-size: 20px;
+  background-color: antiquewhite;
+}
+
+.name-container{
+  margin: 40px;
+}
+.name-container p{
+  font-size: 40px;
+  padding-bottom: 20px;
+  
+ 
+}
+.nextQuestion{
+  border-radius: 10px;
+  padding: 5px;
+  width: 150px;
+  font-size: 20px;
+  background-color: antiquewhite;
+}
+
+.whatDoYou{
+  margin: 40px;
+}
+.whatDoYou h2{
+  font-size: 40px;
+  padding-bottom: 20px;
+  text-align: center;
+  
+}
+.dropMenu{
+  border-radius: 10px;
+  padding: 5px;
+  width: 200px;
+  font-size: 20px;
+  background-color: antiquewhite;
+}
+
+.color{
+  margin: 40px;
+}
+.color h2{
+  font-size: 35px;
+  padding-bottom: 20px;
+  text-align: center;
+  
+}
+
+.favouriteColor{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+.pink{
+ 
+  background-color: #FF007F;
+  border: white solid 2px;
+  width: 100px;
+  height: 100px;
+  margin: 10px;
+  grid-column: 1/2;
+ 
+  }
+
+.black{
+ 
+    background-color: black;
+    border: white solid 2px;
+    width: 100px;
+    height: 100px;
+    padding: 5px;
+    margin: 10px;
+
+}
+
+.answer{
+  padding-left: 30px;
+}
+
+
+.radioButton{
+  margin: 40px;
+}
+.radioButton h2{
+  font-size: 40px;
+  padding-bottom: 20px;
+  text-align: center;
+  
+}
```

Comment:
Take a look at the indentation in the CSS-file and use consistent spacing and indentation.